packages/documentation/copy/en/get-started/TS for Functional Programmers.md

```diff
@@ -593,5 +593,5 @@ consistently used in TypeScript code.
 
 This doc is a high level overview of the syntax and types you would use in everyday code. From here you should:
 
-- Read the full Handbook [from start to finish](/docs/handbook/intro.html)
+- Read the full Handbook [from start to finish](/docs/handbook/2/basic-types.html)
```

Comment:
I think that if anything, we should explicitly state that this is going to skip ahead in the handbook, like the intro page itself. Something like:

> Get started with The Basics

Which is the beginning of the final section of the intro.